plugins/migration/bin/rename.js

```diff
@@ -246,6 +256,18 @@ class VersionRenamer {
   rename(str) {
     for (const entry of this.renamings_) {
       if (str.startsWith(entry.old)) {
+        if (entry.get || entry.set) {
+          console.log(`NOTE: ${entry.old} has been removed.`);
+          if (entry.get) {
+            console.log(`    - Call ${entry.get}() instead of reading it.`);
+          }
+          if (entry.set) {
+            console.log(`    - Call ${entry.set}(/* new value */) instead of ` +
+                'setting it.');
+          }
+          console.log('You will need to manually verify this update.');
+          return (entry.get ?? entry.set) + '()' + str.slice(entry.old.length);
```

Comment:
When I was adding the `-i` option I learned that `console.log` outputs to stdout. I'm not sure if that is desirable in this case. Eg people could be doing `npx @blockly/migrate rename 6 7 my.js > new.js` and they will get these logs in their new.js file.

Reply:
I agree that that is not desirable.  We should use `process.stderr.write` or the like instead.